src/patternfly/components/Menu/menu.scss

```diff
@@ -0,0 +1,252 @@
+  --pf-c-menu__item-description--Color: var(--pf-global--Color--200);
+
+  // icon
+  --pf-c-menu__item-icon--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-menu__item-icon--Color: var(--pf-global--Color--dark-200);
+
+  // toggle icon
+  --pf-c-menu__item-toggle-icon--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-menu__item-toggle-icon--PaddingLeft: var(--pf-global--spacer--sm);
+
+  // select icon
+  --pf-c-menu__item-select-icon--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-menu__item-select-icon--Color: var(--pf-global--active-color--100);
+  --pf-c-menu__item-select-icon--FontSize: var(--pf-global--icon--FontSize--sm);
+
+  // external icon
+  --pf-c-menu__item-external-icon--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-menu__item-external-icon--Color: var(--pf-global--link--Color);
+  --pf-c-menu__item-external-icon--FontSize: var(--pf-global--icon--FontSize--sm);
+
+  // action
+  --pf-c-menu__item-action--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-menu__item-action--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-menu__item-action--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-menu__item-action--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-menu__item-action-icon--Color: var(--pf-global--Color--dark-200);
+  --pf-c-menu__item-action-icon--Height: calc(var(--pf-c-menu__item--FontSize) * var(--pf-c-menu__item--LineHeight));
+  --pf-c-menu__item-action--hover__icon--Color: var(--pf-global--Color--dark-100);
+  --pf-c-menu__item-action--m-favorite__icon--Color: var(--pf-global--disabled-color--200);
+  --pf-c-menu__item-action--m-favorite__icon--FontSize: var(--pf-global--icon--FontSize--sm);
+  --pf-c-menu__item-action--m-favorite--m-favorited__icon--Color: var(--pf-global--palette--gold-400);
+
+  padding-top: var(--pf-c-menu--PaddingTop);
+  padding-bottom: var(--pf-c-menu--PaddingBottom);
+  background-color: var(--pf-c-menu--BackgroundColor);
+  box-shadow: var(--pf-c-menu--BoxShadow);
+
+  &.pf-m-submenu {
+    position: absolute;
+    top: 0;
+    left: 100%;
+  }
+
+  .pf-c-divider {
+    margin-top: var(--pf-c-menu--c-divider--MarginTop);
+    margin-bottom: var(--pf-c-menu--c-divider--MarginBottom);
+  }
+}
+
+// Header
+.pf-c-menu__header {
+  padding-top: var(--pf-c-menu__header--PaddingTop);
+  padding-right: var(--pf-c-menu__header--PaddingRight);
+  padding-bottom: var(--pf-c-menu__header--PaddingBottom);
+  padding-left: var(--pf-c-menu__header--PaddingLeft);
+
+  .pf-c-menu__item {
+    --pf-c-menu__item--PaddingTop: 0;
+    --pf-c-menu__item--PaddingRight: 0;
+    --pf-c-menu__item--PaddingBottom: 0;
+    --pf-c-menu__item--PaddingLeft: 0;
+  }
+}
+
+// List item
+.pf-c-menu__list-item {
+  position: relative;
+  display: flex;
+  color: var(--pf-c-menu__list-item--Color);
+  background-color: var(--pf-c-menu__list-item--BackgroundColor);
+
+  &:hover:not(.pf-m-disabled),
+  &:focus:not(.pf-m-disabled) {
```

Comment:
I didn't see any `tabindex`es on this element so you can't focus on it. Focus styles aren't changing the background color. This may be a use cases for `:focus-within`

Reply:
nice! that works